### DISCUSSION TITLE
仿 Chrome Webstore 风格的论坛类页面

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1596] [插件说明]
描述
---

> 在论坛类页面提供类似 **Chrome Webstore 风格** 主题

地址
---

[https://simpread.ksria.cn/plugins/details/DxlFcL52iy](https://simpread.ksria.cn/plugins/details/DxlFcL52iy)

截图
---

![image](https://user-images.githubusercontent.com/81074/86083467-eb2e5b80-bacc-11ea-89b1-34e381f44ffe.png)

**Comment by FangXzhong** [comment]
您好，我安装了这个插件，但是知乎的样式还是没有改变，想问一下是需要在哪里开启或者设置吗？谢谢！

**Reply by Kenshin** [reply]
并没有问题，安装后随便找个页面 e.g. https://www.zhihu.com/question/266836039/answer/2210940465 就能看到效果了。

![image](https://github.com/Kenshin/simpread/assets/81074/16258e67-c30e-4957-90a2-84608e570c9e)

不启用的话，效果如下

<img width="707" alt="image" src="https://github.com/Kenshin/simpread/assets/81074/a56373c2-b38d-47b6-a640-6db62e98b130">

**Reply by Kenshin** [reply]
@FangXzhong 

请不要通过 Email 回复，因为我看不到图片，你可以在 https://github.com/Kenshin/simpread/discussions/1596 回复我。

**Reply by FangXzhong** [reply]
![image](https://github.com/Kenshin/simpread/assets/64191471/a0aee801-b81b-46b9-8676-d2c43fecfc98)
您好，我安装了插件打开一个知乎页面是这样子的，我用的是同步助手方案，两台windows电脑都是这样，麻烦您啦

**Reply by Kenshin** [reply]
我明白你的问题了，因为使用了自动同步功能导致的，你可以使用 https://www.yuque.com/kenshin/simpread/fcw60u 安装方案。

**Comment by FangXzhong** [comment]
啊，我按照您给的方法成功解决了问题，太谢谢您了！您很耐心，感谢感谢！

**Reply by Kenshin** [reply]
哈，小事情，搞定就好。
